index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <div class="right-side">
+                <a href="#"><h3>Anicet Fantômas</h3></a> 
+            </div>
```

Comment:
- [ ] [ OPTIONAL ]Good work so for by implementing changes required by the previous review! but to be more professional we need to use best practices. Kindly remove the `h3` tag inside the `a` tag
Heading elements can't be childs of anchor elements. you can read it [here](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)